README.md

```diff
@@ -26,6 +26,7 @@ Keybinding                              | Description
 <kbd>Ctrl+K</kbd>, <kbd>Shift+Tab</kbd> | Previous view.
 <kbd>Ctlr+J</kbd>, <kbd>Tab</kbd>       | Next view.
 <kbd>Ctrl+H</kbd>, <kbd>Alt+H</kbd>     | Toggle history.
+<kbd>Ctrl+E</kbd>                       | Export as cURL command
```

Comment:
It's a valid point, but since ctrl+a and ctrl+e don't work in any of the views at the moment I arbitrarily went for ctrl+e :)